lib/java_buildpack/container/tomee/tomee_instance.rb

```diff
@@ -27,9 +27,15 @@ class TomeeInstance < TomcatInstance
       def compile
         download(@version, @uri) { |file| expand file }
         link_to(@application.root.children, root)
-        unless ear?
-          @droplet.additional_libraries << tomcat_datasource_jar if tomcat_datasource_jar.exist?
+        if ear?
+          # if there is a drivers subfolder in the ear app, then link the libraries over to tomee/lib folder
+          if drivers?
+            link_to((@application.root + 'drivers').children, web_inf_lib)
```

Comment:
Yes, was thinking of that @violetagg, however it appears to impact some of the drivers download and linking from what I could see earlier as `web_inf_lib` is an overridden method from the base Tomcat instance.